shell/config/types.js

```diff
@@ -306,3 +306,9 @@ export const AUTH_TYPE = {
   _SSH:   '_ssh',
   _S3:    '_S3'
 };
+
+export const WORKLOAD_TABS = {
```

Comment:
I am sorry if I wasn't clear enough on my explanation @Shavindra , but what I meant is to include these strings on the `en-us` yaml, like we do for all translatable strings like these ones 🙏 🙏 🙏  sorry for that